tests/gnrc_rpl/Makefile

```diff
@@ -9,16 +9,15 @@ USEMODULE += gnrc_rpl
 USEMODULE += shell
 USEMODULE += shell_cmds_default
 
-# automated test only works on native
-TEST_ON_CI_WHITELIST += native
-
 ifeq (native, $(BOARD))
   USEMODULE += socket_zep
   USEMODULE += socket_zep_hello
   USEMODULE += netdev
   TERMFLAGS += -z 127.0.0.1:17754 # Murdock has no IPv6 support
 else
   USEMODULE += netdev_default
+  # automated test only works on native
+  TESTS=
 endif
```

Comment:
A bit out of scope but I'm wondering if that shouldn't go in a `Makefile.board.dep` and if maybe it could also fix the initial issue ?

Reply:
Probably, yes. But there are other tests (e.g. `tests/lwip`) that do this, so definitely out of scope, IMHO. Should I just move the TESTS assignment there for now?

Reply:
Let's do it this way

Reply:
Which way now? :sweat_smile: The way I do it here already or putting it into `Makefile.board.dep`?